app/api/sync/syncWorker.js

```diff
@@ -67,7 +67,7 @@ export default {
   },
 
   async start(interval) {
-    const { sync } = await settings.get();
+    const { sync } = await settings.get(true);
```

Comment:
Should we start a coding practice to pass semantic objects rather than this standalone values?

For example, if this would be: `settings.get({ selectPrivateValues: true})`, I believe this would much more readable.

I also think it helps passing this 'config' arguments in a much better way, if you latter on need a new param, you just add the key, instead of needing to change the number of arguments on the function (but this is kind of secondary to the main legibility point).

What do you guys think? cc @daneryl @konzz @gabriel-piles @fnocetti 
Maybe this is worth discussing in the tech call?

Reply:
@RafaPolit i agree, we use the string notation '+field' but we should use maybe the object notation also supported,
{ field: 1, noReturnField: 0}, as for other scenarios, we can rely on Typescript to know what things are.

Reply:
@RafaPolit ill change it to be more readable.

Reply:
@RafaPolit i did change the settings get method to be more standard, now you actually have to ask for properties that are marked as select: false, explicitly.